tests/stdlib/tmonotimes.nim

```diff
@@ -4,17 +4,33 @@ discard """
 
 import std/[monotimes, times]
 
-let d = initDuration(nanoseconds = 10)
-let t1 = getMonoTime()
-let t2 = t1 + d
+template main =
```

Comment:
The existing test was fine and to the point, your improvements to the number generator seed have nothing to do with this test anyway. Please undo, we cannot keep adding tests that are fine because "under ~ 1 sec".

Reply:
I've removed the loop and changed it to this:
```nim
  block: # getMonoTime is non-decreasing
    let a = getMonoTime()
    let b = getMonoTime()
    doAssert b >= a
```